File: src/com/anathema_roguelike/environment/terrain/grounds/Ground.java
package com.anathema_roguelike.environment.terrain.grounds;

import com.anathema_roguelike.environment.Environment;
import com.anathema_roguelike.environment.Point;
import com.anathema_roguelike.environment.terrain.Terrain;
import com.anathema_roguelike.main.display.VisualRepresentation;

public abstract class Ground extends Terrain {

	public Ground(Environment environment, Point position, char representation, double opacity, double damping) {
		super(environment, position, representation, true, true, opacity, damping);
	}
	
	public Ground(Environment environment, Point position, VisualRepresentation representation, double opacity, double damping) {
		super(environment, position, representation, true, true, opacity, damping);
	}
}

File: src/com/anathema_roguelike/environment/terrain/grounds/Ground.scala
package com.anathema_roguelike
package environment.terrain.grounds

import com.anathema_roguelike.environment.terrain.Terrain
import com.anathema_roguelike.environment.Location
import com.anathema_roguelike.main.display.VisualRepresentation

abstract class Ground(location: Location, representation: VisualRepresentation, opacity: Double, damping: Double)
  extends Terrain(location, representation, true, true, opacity, damping) {
}
